src/main/java/seedu/address/model/client/Timezone.java

```diff
@@ -41,11 +53,9 @@
     public Timezone(String timezone) {
         requireNonNull(timezone);
         checkArgument(isValidTimezone(timezone), MESSAGE_CONSTRAINTS);
-        String offsetString = timezone.substring(GMT_STRING.length());
-        int offsetValue = Integer.parseInt(offsetString);
+        assert (VALID_TIMEZONES.contains(timezone.substring(UTC_STRING.length())));
```

Comment:
```suggestion
        assert VALID_TIMEZONES.contains(timezone.substring(UTC_STRING.length()));
```
btw what does this assert do? Isit rly needed lmao